interpreter/cling/lib/Interpreter/ValueExtractionSynthesizer.cpp

```diff
@@ -420,7 +421,10 @@ namespace {
     R.clear();
     R.setLookupName(&m_Context->Idents.get("copyArray"));
     m_Sema->LookupQualifiedName(R, NSD);
-    assert(!R.empty() && "Cannot find cling::runtime::internal::copyArray");
+    /* If I am not being called from a child Interpreter, only then do
```

Comment:
Well - this is a temporary hack until we can import copyArray. And once we can import it, this member and the assert-protection should go away.

Could you clarify that in the comment, please?